Sources/Stevenson/SlackService.swift

```diff
@@ -78,10 +80,23 @@ public struct SlackService {
             return try SlackResponse(command.help)
                 .encode(for: request)
         } else {
-            return try command
-                .run(metadata, request)
-                .mapIfError { SlackResponse($0.localizedDescription, visibility: .user) }
+            return try SlackResponse("Ok!")
```

Comment:
We should try to reply with a more explicit response depending on the command to show that we understood the request properly. For example for a fastlane command we could reply "Ok, I'm gonna trigger the lane X on branch Y right away!"

Reply:
This then will need to be defined in the command, I'd prefer it to be simple if possible and just give some generic response. Alternatively we can wrap this in a function `SlackService.replyLater` so that commands can use it on their own.